Automattic/wp-calypso
# Move domain search suggestion onClick to Select Button

Fixes https://github.com/Automattic/wp-calypso/issues/94934

## Proposed Changes

* Moves domain suggestion onClick handler from the result row div to the domain select button.

![Screenshot(103)](https://github.com/user-attachments/assets/82327766-bc39-4eb9-b815-5b700bac8018)

## Testing Instructions

* Search for a new domain in the domains add flow http://calypso.localhost:3000/domains/add
* Clicking on any domain information should not select the domain
* Clicking on the select domain button should move you to the email upsell
* Tracks event should continue to fire with the same information
* Run in console: `localStorage.setItem( 'debug', 'calypso:analytics' );`

Repeat testing for the domain only flow: http://calypso.localhost:3000/start/domain/domain-only

Before
![Screenshot(100)](https://github.com/user-attachments/assets/d1fca55f-36e8-4465-9d71-62d5cadf5e46)

After

![Screenshot_2024-11-04_12-57-54](https://github.com/user-attachments/assets/464015c6-617f-466f-9ca0-0a6e5df2f211)
